quinn-proto/src/transport_parameters.rs

```diff
@@ -203,7 +217,7 @@ impl From<UnexpectedEnd> for Error {
 }
 
 impl TransportParameters {
-    pub fn write<W: BufMut>(&self, w: &mut W) {
+    pub(crate) fn write<W: BufMut>(&self, w: &mut W) {
```

Comment:
I think methods to read from `&[u8]`/write to a `Vec<u8>` need to be exposed for the crypto traits to actually be implementable.

Reply:
You're right, of course. Fixed.